superset/utils/pandas_postprocessing/flatten.py

```diff
@@ -25,12 +27,14 @@
 def flatten(
     df: pd.DataFrame,
     reset_index: bool = True,
+    drop_levels: Union[Tuple[int, ...], Tuple[str, ...]] = (),
```

Comment:
nit: since I assume we'll be getting `List` (not `Tuple`) here due to the args coming from a JS object, and `Dataframe.droplevel` [works with list-like objects](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.droplevel.html#pandas.DataFrame.droplevel), maybe it would be more appropriate to use something like

```suggestion
    drop_levels: Optional[Union[Sequence[int, ...], Sequence[str, ...]]] = None,
```

Reply:
thanks! I will update.